src/swarm-tui/Swarm/TUI/View.hs

```diff
@@ -1060,27 +1074,56 @@ drawKeyMenu s =
     [ ("↓↑", "history")
     ]
       ++ [("Enter", "execute") | not isReplWorking]
-      ++ [("^c", "cancel") | isReplWorking]
-      ++ [("M-p", renderPilotModeSwitch ctrlMode) | creative]
-      ++ [("M-k", renderHandlerModeSwitch ctrlMode) | handlerInstalled]
+      ++ [(keyR SE.CancelRunningProgramEvent, "cancel") | isReplWorking]
+      ++ [(keyR SE.TogglePilotingModeEvent, renderPilotModeSwitch ctrlMode) | creative]
+      ++ [(keyR SE.ToggleCustomKeyHandlingEvent, renderHandlerModeSwitch ctrlMode) | handlerInstalled]
       ++ [("PgUp/Dn", "scroll")]
   keyCmdsFor (Just (FocusablePanel WorldPanel)) =
-    [ ("←↓↑→ / hjkl", "scroll") | canScroll
+    [ ( T.intercalate "/" $
+          map
+            keyW
+            [ SE.MoveViewWestEvent
+            , SE.MoveViewNorthEvent
+            , SE.MoveViewEastEvent
+            , SE.MoveViewSouthEvent
+            ]
```

Comment:
Code like this can be simplified with `enumerate` after refactoring to use the `MoveViewDirectionEvent AbsoluteDir`